integrations/mysql/documentation.yaml

```diff
@@ -7,47 +7,46 @@ exporter_name: the MySQL exporter
 exporter_pkg_name: mysqld_exporter
 exporter_repo_url: https://github.com/prometheus/mysqld_exporter
 dashboard_available: true
-minimum_exporter_version: v0.14.0
+minimum_exporter_version: v0.15.0
 multiple_dashboards: false
 dashboard_display_name: {{app_name_short}} Prometheus Overview
 config_mods: |
-  apiVersion: apps/v1
```

Comment:
As per the other example, we can update https://github.com/GoogleCloudPlatform/prometheus-engine/blob/main/examples/mysql/exporter.yaml.snippet directly (after addressing the other comments here)

But unlike the consul example, this PR doesn't become completely obsolete -- we do still need to update this file because of the change you made in line 70 (additional_install_info).

Reply:
Sounds good, just for clarity I wanted to make sure that the line you're mentioning here doesn't need to be changed? We just want to update things in this repo and the other one, right?

Reply:
Yep